cdci_data_analysis/analysis/instrument.py

```diff
@@ -105,6 +118,8 @@ def __init__(self,
         self.input_product_query=input_product_query
 
         self.query_dictionary = query_dictionary
+        
+        self.allow_unknown_arguments = allow_unknown_arguments
```

Comment:
we have to write somewhere in a comment what's the difference between `argument` and `parameter`